src/util.h

```diff
@@ -115,6 +116,12 @@ bool Truncate(const string& path, size_t size, string* err);
 #endif
 
 #ifdef _WIN32
+/// UTF-8 converters
+std::wstring Utf8ToWide(const StringPiece& s);
```

Comment:
StringPiece and WStringPiece should always be passed by value, never by const-reference.